Themes/Ecuador/AndroidManifest.xml

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (c) 2010 Ushahidi Inc All rights reserved Contact: team@ushahidi.com 
+	Website: http://www.ushahidi.com GNU Lesser General Public License Usage 
+	This file may be used under the terms of the GNU Lesser General Public License 
+	version 3 as published by the Free Software Foundation and appearing in the 
+	file LICENSE.LGPL included in the packaging of this file. Please review the 
+	following information to ensure the GNU Lesser General Public License version 
+	3 requirements will be met: http://www.gnu.org/licenses/lgpl.html. If you 
+	have questions regarding the use of this file, please contact Ushahidi developers 
+	at team@ushahidi.com.
+-->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.ec.ushahidi.android.app"
```

Comment:
Marica como la Tere la loca del pixel a ti te voy a llamar la loca del dominio, jaja

Pablo
> @@ -0,0 +1,317 @@
> +<?xml version="1.0" encoding="utf-8"?>
> +<!--
> +     Copyright (c) 2010 Ushahidi Inc All rights reserved Contact: team@ushahidi.com
> +     Website: http://www.ushahidi.com GNU Lesser General Public License Usage
> +     This file may be used under the terms of the GNU Lesser General Public License
> +     version 3 as published by the Free Software Foundation and appearing in the
> +     file LICENSE.LGPL included in the packaging of this file. Please review the
> +     following information to ensure the GNU Lesser General Public License version
> +     3 requirements will be met: http://www.gnu.org/licenses/lgpl.html. If you
> +     have questions regarding the use of this file, please contact Ushahidi developers
> +     at team@ushahidi.com.
> +-->
> +<manifest xmlns:android="http://schemas.android.com/apk/res/android"
> +    package="org.ec.ushahidi.android.app"

Cambiar a "ec.desastre.android.app"

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/desastre-ecuador/Ushahidi_Android/pull/5/files/5796ea4924b5bad19cd4725056f9789a61966b0d#r60956564